runners/core-java/src/main/java/org/apache/beam/runners/core/SplittableParDo.java

```diff
@@ -284,7 +284,7 @@ public PCollectionTuple expand(
       return input
           .getPipeline()
           .getCoderRegistry()
-          .getDefaultCoder(output.getTypeDescriptor(), fn.getInputTypeDescriptor(), inputCoder);
+          .getCoder(output.getTypeDescriptor(), fn.getInputTypeDescriptor(), inputCoder);
```

Comment:
Separate the rename of `getDefaultCoder` -> `getCoder` into a commit, so it can be ignored for real review.